src/entities/addLiquidity/index.ts

```diff
@@ -33,6 +33,13 @@ export class AddLiquidity implements AddLiquidityBase {
     }
 
     buildCall(input: AddLiquidityCall): AddLiquidityBuildOutput {
+        // TODO: refactor validators to take v3 into account
```

Comment:
Is this being tracked in a new issue?

Reply:
I just created #280 to keep track of this ✅